docs/guides/migrations.md

```diff
@@ -0,0 +1,83 @@
+});
+
+keystone.createList('ExampleList', {
+  fields: {
+    age: { type: Integer },
+  },
+});
+```
+
+Using the KnexAdapter if we create an `ExampleList` that contains an `Integer` field called `age` when we start Keystone for the first time it will generate a table with an `id` and `age` column:
+
+| table_name  | column_name | data_type |
+| ----------- | ----------- | --------- |
+| ExampleList | id          | integer   |
+| ExampleList | age         | integer   |
+
+If we update our Keystone List either by renaming fields or adding new fields, we might encounter errors next time we start Keystone. This is because the database schema has changed and Keystone doesn't know what to do with existing data or won't be able to save new data. Let's imagine that because we are responsible with data collection we want to update the list to only include an age range rather than a specific age. We might update the field to use a select type:
+
+```javascript
+keystone.createList('MigrationExample', {
+  fields: {
+    age: { type: Select, options: '0-18, 19-35, 36-50, 50+' },
+  },
+});
+```
+
+If we restart Keystone after this change it will throw an error. The `data_type` of the `age` column is now a `text` value and it should contain a string representation of the age range. The table should be change to something like the following:
+
+| table_name  | column_name | data_type |
+| ----------- | ----------- | --------- |
+| ExampleList | id          | integer   |
+| ExampleList | age         | text      |
+
+Keystone cannot make this change for you because there may be existing data with a type of `integer`. Sometimes migration is as simple as converting an `integer` to a `string` but in certian cases, such as this, it requires programatically resolving the appropriate.
+
+## Migration options
+
+In development sometimes it can be helpful to drop the database everytime Keystone is restarted. To do this using the `KnexAdapter` you can pass the `dropDatabase` option to the adapter:
+
+```js
+const { KnexAdapter } = require('@keystone-alpha/adapter-knex');
+const adapter = new KnexAdapter({ dropDatabase: true });
+```
+
+This might be enough for small projects but it won't be ideal for everyone. If you want data to persist in development you need to re-initialise the database or preform migrations. And regardless of the development environment, almost all projects will eventually need to make changes to a List table in production. For this there are a number of solutions.
+
+The most basic solution is to make these changes on the database. You can do this via a command-line, but if you're reading this guide you're probably looking for an application to help you view and manage tables created by Keystone. For Mongo you might try (Studio 3T)[https://studio3t.com/download/] which is free for non-commercial use. For Postgres you can try (pgadmin)[https://www.pgadmin.org/]. There are a lot of other options and you should find one that works for you.
```

Comment:
That "if you're reading this" could be interpreted in condescending ways.  How about this?  (Also some link fixes.)

```suggestion
The most basic solution is to make these changes on the database. You can do this via a command-line, or you might prefer a graphical DB admin interface. For Mongo you might try [Studio 3T](https://studio3t.com/download/) which is free for non-commercial use. For Postgres you can try [pgadmin](https://www.pgadmin.org/). There are a lot of other options and you should find one that works for you.
```